packages/shared/src/underscore.ts

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
I moved these helper utils into an "underscore" file (like `_`/lodash, you see :P)